tests/app/forms/field_handlers/test_text_area_handler.py

```diff
@@ -73,3 +73,24 @@ def test_get_length_validator_with_max_length_override():
     validator = text_area_handler.get_length_validator()
 
     assert validator.max == 30
+
+
+def test_get_textarea_row_length():
```

Comment:
- Seems odd to use length, `test_get_textarea_rows` would be better.
- We should have another test to test the default rows i.e. no rows attribute in `answer`.